libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -1721,14 +1722,21 @@ bool AP_AHRS_NavEKF::have_ekf_logging(void) const
     return false;
 }
 
-// get the index of the current primary IMU
-uint8_t AP_AHRS_NavEKF::get_primary_IMU_index() const
+// get earth-frame accel vector for primary IMU
+const Vector3f &AP_AHRS_NavEKF::get_accel_ef() const
```

Comment:
minor niggle, if you leave get_accel_ef() below get_primary_accel_index() patch will look smaller and be easier to review